oaknational/Oak-Web-Application
# Feat/connect new plan a lesson cms - LESQ-814

## Description

Music year: {owa_music_year}

- [x]  Add zod and types for new schema in OWA
- [x]  Add GQL for new page
- [x]  Pull data from sanity into page 
- [x] updates test and updates fixtures


## How to test

1. Go to https://deploy-preview-2408--oak-web-application.netlify.thenational.academy/lesson-planning-new
Open console - you should see test page data coming through from sanity

## Checklist
